test/unit/test-custom-element.js

```diff
@@ -641,6 +656,101 @@ describes.realWin('CustomElement', {amp: true}, (env) => {
         );
       });
 
+      describe('granular consent experiment', () => {
+        beforeEach(() => {
+          toggleExperiment(win, 'amp-consent-granular-consent', true);
+        });
+
+        afterEach(() => {
+          toggleExperiment(win, 'amp-consent-granular-consent', false);
```

Comment:
FMI: do we explicitly need to toggle experiments off or are they reset each time anyway?

Reply:
Explicit is better to avoid memory leaks.

Reply:
Any particular reason for doing `beforeEach/afterEach` instead of `beforeAll/afterAll`?

Reply:
We are not allowed to use `beforeAll/afterAll`